packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -169,14 +183,10 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
 
     // 3. Check responses.
     if (!priceResponse || !priceResponse.result || !priceResponse.result.price) {
-      // TODO: Slack transport might not be able to handle long messages such as the stringified `ohlcResponse`,
```

Comment:
Unrelated to PR but removed as addressed in PR: https://github.com/UMAprotocol/protocol/pull/2124